src/etcd/client.py

```diff
@@ -784,6 +784,7 @@ def api_execute(self, path, method, params=None, timeout=None):
                               and cluster_id is not None and
                               cluster_id != self.expected_cluster_id)
                 # Update the ID so we only raise the exception once.
+                old_expected_cluster_id = self.expected_cluster_id
```

Comment:
just the self.expected_cluster_id, but no we can't, I see it now.

LGTM

On Mon, Jun 22, 2015 at 5:09 PM, Alex Chan <notifications@github.com> wrote:

> In src/etcd/client.py
> <https://github.com/jplana/python-etcd/pull/113#discussion_r32944569>:
>
> > @@ -784,6 +784,7 @@ def api_execute(self, path, method, params=None, timeout=None):
> >                                and cluster_id is not None and
> >                                cluster_id != self.expected_cluster_id)
> >                  # Update the ID so we only raise the exception once.
> > +                old_expected_cluster_id = self.expected_cluster_id
>
> Do you mean the assignment of old_expected_cluster_id,
> self.expected_cluster_id, or both?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jplana/python-etcd/pull/113/files#r32944569>.
>